apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/start/en.ftl

```diff
@@ -2,3 +2,8 @@
 next-new-user-step-1-2 = 1. Create a { -product-mozilla-account }
 next-new-user-sign-in-link-2 = Already have a { -product-mozilla-account }? <a>Sign in</a>
 next-payment-confirm-with-legal-links-static-3 = I authorize { -brand-mozilla } to charge my payment method for the amount shown, according to <termsOfServiceLink>Terms of Service</termsOfServiceLink> and <privacyNoticeLink>Privacy Notice</privacyNoticeLink>, until I cancel my subscription.
+
+next-payment-method-header = Choose your payment method
+# This message is used to indicate the second step in a multi step process.
+payment-method-header-second-step-next = 2. { next-payment-method-header }
```

Comment:
L10n review note, although this is mostly a reused string (Which we've decided to prefix with `next-` in most typical cases), since the value of it changes compared to the string currently in use I've decided to create a new string Id for it.